cmd/workspace/workspace/overrides.go

```diff
@@ -17,6 +22,48 @@ func listOverride(listCmd *cobra.Command, listReq *workspace.ListWorkspaceReques
 func exportOverride(exportCmd *cobra.Command, exportReq *workspace.ExportRequest) {
 	// The export command prints the contents of the file to stdout by default.
 	exportCmd.Annotations["template"] = `{{.Content | b64_decode}}`
+	exportCmd.Use = "export SOURCE_PATH"
+
+	var filePath string
+	exportCmd.Flags().StringVar(&filePath, "file", "", `Path on the local file system to save exported file at.`)
+
+	exportCmd.RunE = func(cmd *cobra.Command, args []string) error {
+		ctx := cmd.Context()
+		w := root.WorkspaceClient(ctx)
+
+		if len(args) == 0 {
+			promptSpinner := cmdio.Spinner(ctx)
+			promptSpinner <- "No SOURCE_PATH argument specified. Loading names for Workspace drop-down."
+			names, err := w.Workspace.ObjectInfoPathToObjectIdMap(ctx, workspace.ListWorkspaceRequest{})
```

Comment:
Does the dropdown actually work? I think to make it work we have to provide `Path` field. And even if we do, are we going default to "/"? Is it really useful then?

Reply:
This code as been copy-pasted from the autogenerated version (https://github.com/databricks/cli/blob/9b6a8471786a2d1c7d0a3911dfba00d7e981ab2d/cmd/workspace/workspace/workspace.go#L167). This was the best way to add the `--file` flag. 

You make a good point about the prompting though. I tried it out and it does not work. Here's a sample output:
```
databricks workspace export
⣾ No PATH argument specified. Loading names for Workspace drop-down.Error: failed to load names for Workspace drop-down. Please manually specify required arguments. Original error: Path () doesn't start with '/'
```

I think we can remove the prompting logic entirely. I have filed a followup in our internal backlog to do so.

Reply:
If it doesn't work, please remove it, there is no reason to keep it in override then